libpod/runtime.go

```diff
@@ -96,6 +96,17 @@ func NewRuntime(options ...RuntimeOption) (*Runtime, error) {
 	return runtime, nil
 }
 
+// GetStore returns the store used by the runtime
+func (r *Runtime) GetStore() storage.Store {
```

Comment:
Really don't like this. Where do we need it? Trying really hard not to expose the store directly in libpod.

Reply:
I don't think we need it, comes from the original version.  Perhaps leftover cruft.

Reply:
fixed